website/docs/addon_applications_admin.md

```diff
@@ -1,26 +1,235 @@
+
+
+![](assets/applications/tools_filters_profiles.png)
+
+1. **Folder Paths:** Create a list of folder paths to determine which ones the filter profile will affect. *It supports regex patterns.*
+2. **Task Types:** Choose from a list of task types to determine which ones the filter profile will affect.
+3. **Task Names:** Create a list of task names to determine which ones the filter profile will affect.
+4. **Tools:** Tools list to determine which ones the filter profile will affect.
+5. **+ button:** To add more applications filters profiles.
+
+:::tip Example Filter profile
+Find example of Tool defintion and a Filter profile in the FAQ section below:
+[How to make a tool available for all folder paths in a project for a specific application variant ?](addon_applications_admin#how-to-make-a-tool-available-for-all-folder-paths-in-a-project-for-a-specific-application-variant-)
+:::
+
+## FAQ
+### How to use application variant group settings ?
+
+<details><summary>How to use application variant group settings ?</summary>
+
+Variant group settings is used to add more versions of an application.
+There are two ways of doing it.
+
+1. **Add new executable** to an existing application version. This is a good way if you have multiple fully compatible versions of your DCC across the studio. Nuke is a typical example where multiple artists might have different `v#` releases of the same minor Nuke release. For example `12.2v3` and `12.3v6`. When you add both to `12.2` Nuke executables they will be treated the same in AYON and the system will automatically pick the first that it finds on an artist machine when launching. Their order is also the order of their priority when choosing which version to run if multiple are present.
+![settings_applications](assets/applications/settings_addapplication.gif)
+
+2. **Add version** in case you want this version to be selectable individually. This is typically used for bigger releases that might not be fully compatible with previous versions. Keep in mind that if you add the latest version of an Application that is not yet part of the official AYON release, you might run into problems with integration. We test all the new software versions for compatibility and most often, smaller or bigger updates to AYON code are necessary to keep everything running.
+![settings_applications](assets/applications/settings_addappversion.gif)
+
+</details>
+
+### What is host name used for ?
+
+<details><summary>What is host name used for ?</summary>
+
+AYON uses host name to match applications to their host integration addons.
+different applications can have the same host name as long as they are related to the same addon.
+And, host name is defined in the addon's client code. e.g. [nuke's addon host name](https://github.com/ynput/ayon-nuke/blob/7a08f570512d5bd92caf014ba159889cfd912d38/client/ayon_nuke/addon.py#L13).
+
+![](assets/applications/different_apps_same_host.png)
+
+:::tip
```

Comment:
This is admin documentation. The explanation is oversimplified (sounds almost magical), I would try to avoid "magical feel" to avoid disapointment.